src/tools/std23/views/stride.hpp

```diff
@@ -336,6 +344,21 @@ struct stride_view : std20::ranges::view_interface< stride_view< R > > {
 template < typename R >
 stride_view( R&&, std20::ranges::range_difference_t< R > ) -> stride_view< std20::ranges::all_view< R > >;
 
+} // namespace ranges
+} // namespace std23
+
+namespace std20 {
+inline namespace ranges {
+
+template <typename R>
+inline constexpr bool enable_borrowed_range<std23::ranges::stride_view<R>> = enable_borrowed_range<R>;
+
+} // namespace ranges
+} // namespace std23
```

Comment:
namespace std23? matches to bracket std20

Reply:
That's indeed a typo